docs/UserGuide.md

```diff
@@ -3,10 +3,23 @@ layout: page
 title: User Guide
 ---
 # TutorBuddy
+Welcome to the user guide for TutorBuddy!<br/>
+Are you a tuition teacher looking for a reliable way to keep track of
+your 1-to-1 classes? Needing a system to remind you about upcoming classes?
+Or efficient way to calculate your class incomes? <br/>
```

Comment:
Hmmm maybe using "class" here might not be suitable as a class is usually associated with a group 1-many tuition. Just putting as "Or efficient way to calculate your incomes?" might be better.